test/compilable/dtoh_StructDeclaration.d

```diff
@@ -70,6 +76,12 @@ struct S2
         c()
     {
     }
+    S2(int32_t a, int32_t b = 0, int64_t c = 0LL, S d = S(0, 0, 0LL, nullptr)) :
+        a(a),
+        b(b),
+        c(c),
+        d(d)
+        {}
```

Comment:
This makes `S2 s{123};` ambiguous due to existing `S2(int32_t)` ctor in line 71.

Reply:
Fixed.